docs/configuration.md

```diff
@@ -111,7 +111,6 @@ Unless you need to set a non-default value, it is recommended to only populate o
 | `REDIS_RECONNECT_DELAY_MS`                  | "1000"                                | Sets the delay between reconnect retries from the Redis client in ms                                                                                                                                                                                                                                   |
 | `SEND_RAW_TRANSACTION_SIZE_LIMIT`           | "131072"                              | Sets the limit of the transaction size the relay accepts on eth_sendRawTransaction                                                                                                                                                                                                                     |
 | `GET_RECORD_DEFAULT_TO_CONSENSUS_NODE`      | "false"                               | Flag to set if get transaction record logic should first query the mirror node (false) or consensus node via the SDK (true).                                                                                                                                                                           |
-| `HBAR_RATE_LIMIT_WHITELIST`                 | [""]                                  | An array of EVM addresses that are allowed to bypass the HBAR rate limits                                                                                                                                                                                                                              |
```

Comment:
Q: Was this ever used?
If so what version was this added in?

Reply:
This is from the old hbar limiter and it's no longer used anywhere, so I removed it

Reply:
Hmmm, we can't remove configs easily cause we may have operators who've adopted it.
Unless it's removal makes no difference to an operator we should leave it in an have a deprecation plan

Reply:
Well, in this case it makes no difference because this configuration is no longer used. 

I guess we should note in the release notes it was deprecated (it was used in the old hbar limiter, but we didn't include support for it in the redesigned version)